src/datasources/notebook/QueryBuilder.tsx

```diff
@@ -162,8 +163,9 @@ export const TestResultsQueryBuilder: React.FC<TestResultsQueryBuilderProps> = (
       messages={queryBuilderMessages}
       showIcons
       // Only set value on first render
-      {...(initialize.current && { value: props.defaultValue })}
+      {...(initialize.current && { value: filterXSSLINQExpression(props.defaultValue) })}
       {...props}
+      value={filterXSSLINQExpression(props.defaultValue)}
```

Comment:
@mure Is there a reason why you haven't been using the `value` attribute in the query builder and instead you were only using the `defaultValue` ?

Reply:
Are you just asking why I named the prop `defaultValue` and not `value`?

I can't remember exactly why, it's been a long time since I touched this code. Probably because it's a convention in React that `defaultValue` sets the initial value for the control. `value` is used for controlled components: https://legacy.reactjs.org/docs/forms.html#controlled-components